src/app/app-routing.module.ts

```diff
@@ -1,9 +1,14 @@
 import { NgModule } from '@angular/core';
-import {RouterModule, Routes} from "@angular/router";
+import { RouterModule, Routes } from "@angular/router";
+import { LandingComponent } from './landing/landing/landing.component';
+import { ProfileComponent } from './profile/profile/profile.component';
 
 const routes: Routes = [
-  { path: '', redirectTo: 'properties', pathMatch: 'full' },
+  { path: '', redirectTo: 'home', pathMatch: 'full' },
+  { path: 'home', component: LandingComponent },
```

Comment:
It would be better if this was lazy loaded with the loadChildren property, but this only applies to performance so it doesn't matter too much right now and can be changed in the future if there is a need for it.